import-spreadsheets.qmd

```diff
@@ -1,11 +1,26 @@
-# Spreadsheets {#import-spreadsheets}
+# Spreadsheets {#sec-import-spreadsheets}
+
+```{r}
+#| label: setup
+#| file: "_common.R"
+#| include: false
+```
+
+::: status
```

Comment:
Is there a reason to not generate the div in `status()`?

Reply:
No reason, this is just one way I figured out to mimic the current styling. I figured these will ultimately go away anyway.